vuetifyjs/vue-cli-plugins
# feat(v3-preset): Add typescript support

TS Support for Vuetify 3 CLI Alpha testing.

### To test

- Clone and check out this branch

In a separate directory
```bash
vue create my-app
```

#### Vue prompt
Manually select Vue 3, TypeScript, Vuex and Vue Router.
```bash
? Please pick a preset: (Use arrow keys)
    Default ([Vue 2] babel, eslint) 
    Default (Vue 3 Preview) ([Vue 3] babel, eslint) 
  ❯ Manually select features 
```

#### cd into app dir
```bash
cd my-app
```

#### Install vuetify/vue-cli-plugins
```bash
yarn add -D '../local/path/to/vue-cli-plugins-vuetify'
```

#### Invoke vuetify
```bash
vue invoke vuetify
```

#### Vuetify prompt
```bash
? Choose a preset: (Use arrow keys)
  Default (recommended) 
  Preview (v3 + vite)
  Prototype (rapid development) 
❯  V3 (alpha) 
  Configure (advanced) 
```

#### Run app
```bash
yarn dev
```
